knpu/ep3b-doctrine-relations/relationship-types.md

```diff
@@ -108,6 +108,13 @@ Yep: `CREATE TABLE tag`. Go run it:
 php bin/console doctrine:migrations:migrate
 ```
 
+***TIP
+If you got an error like "Syntax error or access violation: 1071 Specified key was too long;..." 
+
+No worries! Just open your Entity and modify `@Slug` annotation, set `length=180`. Remove 
+migration file and repeat last 2 steps. Or update MySQL to version 5.7
```

Comment:
It's not the `@Slug` annotation - it's the `@Column` annotation above the `slug` property, right?